clients/vscode-hlasmplugin/schema/proc_grps.schema.json

```diff
@@ -24,15 +24,16 @@
                                     "properties": {
                                         "path": {
                                             "type": "string",
-                                            "description": "Path to the folder containing external files."
+                                            "description": "Path to the folder that contains external files."
                                         },
                                         "optional": {
                                             "type": "boolean",
-                                            "description": "Specifies if it is acceptable not to be able locate the folder."
+                                            "description": "TODO Specifies if it is acceptable not to be able locate the folder."
```

Comment:
It's still hard to comprehend, but let me try to reword the sentence to see if I understand the point of this option. 

"Specifies whether to show an error message if the folder is not located" 

Or am I missing the mark?

Reply:
more like `not to show`....

false - generates error when missing
true - error is suppressed

Reply:
the first variant would need be "not show... not located"... I'd prefer a description that contains no negatives... so that the user does not have to think about "not not conditions" when choosing the option

Reply:
Ok, Then the sentence should read "Specifies whether to not show an error message if the folder is not located", Is that right?

Reply:
yes, but good luck picking the right value... the original wording basically posed a question "is it acceptable not to be able to locate the folder?" If yes, put true, if no, put false.

Reply:
The original wording doesn't imply any errors. What does 'acceptable' mean in that context? One can only guess...

Reply:
ok.... the user defines a list of folders that are used to locate dependencies (macros, copybooks).
The plugin searches these libraries and the default behavior to report an error when this cannot be accomplished (folder does not exist, some kind of failure...).
This option indicates that it is acceptable to the user if the library mentioned in the list does not exist i.e. it is `optional`.